app/src/main/res/values/dimens.xml

```diff
@@ -5,6 +5,7 @@
   <dimen name="fab_margin">16dp</dimen>
   <dimen name="headingText">16sp</dimen>
 
+  <dimen name="cameraIcon">30dp</dimen>
```

Comment:
Still the hardcoded dimensions. This will not solve the problem. Dimensions should be done pro-grammatically. Just a few more lines needed.